docs/guides/esp32/vs_code_development.md

```diff
@@ -0,0 +1,71 @@
+# Visual Studio Code Development
```

Comment:
Should there be an entry for this in: https://github.com/project-chip/connectedhomeip/tree/master/docs/guides/esp32#espressif-esp32-getting-started-guide ?

Reply:
Yes we should add it there